authentication/account/service/user.go

```diff
@@ -114,6 +114,10 @@ func (s *userServiceImpl) BanUser(ctx context.Context, username string) (*reposi
 		return s.Repositories().Users().Save(ctx, &identity.User)
 	})
 
+	if err == nil {
+		err = s.deleteUserFromOtherServices(ctx, identity)
+	}
+
```

Comment:
shall we not call this before DB changes ??  I guess, DB changes is better to be last thing to do.

Reply:
I think we should mark user as banned in Auth DB even if Che or Tenant calls failed.

Reply:
I get your point, @alexeykazakov , but what happens if the user is marked as banned but Che or Tenant deletion failed? We wouldn't be able to process the user again, would we?

Reply:
Yes we should be able to re-ban the user if it failed in Che or Tenant the first time. I added a test for that.
Unlike deactivation we don't change the username or email of the user we are banning. We just set the ban flag. We can do it multiple times.